File: backend/src/main/java/com/objectiveplatform/beerlovers/dto/Beer.java
package com.objectiveplatform.beerlovers.dto;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;

import javax.validation.constraints.Min;
import javax.validation.constraints.NotBlank;

@JsonIgnoreProperties(ignoreUnknown = true)
@Data
public class Beer {
    @Min(0)
    @JsonProperty(value = "id")
    private int id;
    @NotBlank
    @JsonProperty(value = "name")
    private String name;
    @NotBlank
    @JsonProperty(value = "description")
    private String description;
    @NotBlank
    @JsonProperty(value = "tagline")
    private String tagline;
    @JsonProperty(value = "image_url")
    private String imageUrl;
}

File: frontend/src/app/model/beer.ts
export class Beer {
    id: number;
    name: string;
    description: string;
    tagline: string;
    image_url: string;

    constructor(id: number, name: string, 
        description : string, tagline : string, image_url : string) {
        this.id = id;
        this.name = name;
        this.description = description;
        this.tagline = tagline;
        this.image_url = image_url;
      }
}
